### PR TITLE
Add use-unknown-property rule.

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

---
This change is [Reviewable](https://reviewable.io/reviews/polymer/polymer-linter/40)

### CODE REVIEW

**Comment by usergenic** [comment]
Reviewed 21 of 21 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[src/test/polymer/databind-with-unknown-property_test.ts, line 76 at r1](https://reviewable.io:443/reviews/polymer/polymer-linter/40#-Kej1awhkVvVX7PcUuMV:-Kej1awhkVvVX7PcUuMW:b-6czwe2) ([raw file](https://github.com/polymer/polymer-linter/blob/25434177633c8a1647ff173d7bdd97bc7678141f/src/test/polymer/databind-with-unknown-property_test.ts#L76)):*
> ```XML
>           'be a declared property.',
>       'referencedOnlyOnce is not declared or used more than once. ' +
>           'Did you mean: translate'
> ```

It'd be nice to have a better example of the closest word suggestion; this suggestion seems arbitrary.  Just a thought.  Still useful from a test perspective to verify expected output.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/polymer/polymer-linter/40)*

**Comment by usergenic** [comment]
:lgtm:

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/polymer/polymer-linter/40#-:-Kej5RCZy5k6Rwy7qNA4:bnfp4nl)*

**Review by usergenic — APPROVED** [review]
LGTM

"Reviewable" app does not "Approve" using Github's own Review records.  (That was counterintuitive)

File: src/html/util.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
```
Comment (valdrinkoshi) [inline src/html/util.ts]: (c) 2017

Comment (rictic) [reply src/html/util.ts]: Done

**Comment by rictic** [comment]
Review status: 18 of 21 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/test/polymer/databind-with-unknown-property_test.ts, line 76 at r1](https://reviewable.io:443/reviews/polymer/polymer-linter/40#-Kej1awhkVvVX7PcUuMV:-KejqAOTPd-B-eAxB2_1:b-8a71fh) ([raw file](https://github.com/polymer/polymer-linter/blob/25434177633c8a1647ff173d7bdd97bc7678141f/src/test/polymer/databind-with-unknown-property_test.ts#L76)):*
<details><summary><i>Previously, usergenic (Brendan Baldwin) wrote…</i></summary><blockquote>

It'd be nice to have a better example of the closest word suggestion; this suggestion seems arbitrary.  Just a thought.  Still useful from a test perspective to verify expected output.
</blockquote></details>

+1. Added another case to demonstrate that the spelling suggestion is doing something reasonable looking.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/polymer/polymer-linter/40)*

**Comment by rictic** [comment]
Review status: 18 of 21 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/html/util.ts, line 3 at r1](https://reviewable.io:443/reviews/polymer/polymer-linter/40#-Kdmsjs8uE5QNFjHXcIB-r1-3:-KejqjAML35yA96dt7OH:b-896fix) ([raw file](https://github.com/polymer/polymer-linter/blob/25434177633c8a1647ff173d7bdd97bc7678141f/src/html/util.ts#L3)):*
<details><summary><i>Previously, rictic (Peter Burns) wrote…</i></summary><blockquote>

Done
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/polymer/polymer-linter/40)*

**Pull request merged**